homeassistant/components/mikrotik/device_tracker.py

```diff
@@ -32,7 +29,7 @@ async def async_setup_entry(
     """Set up device tracker for Mikrotik component."""
     hub = hass.data[DOMAIN][config_entry.entry_id]
```

Comment:
I'd recommend to rename the variable everywhere to `coordinator`. That's standard.